src/libwalletqt/Wallet.cpp

```diff
@@ -347,6 +352,20 @@ void Wallet::setSubaddressLabel(quint32 accountIndex, quint32 addressIndex, cons
 {
     m_walletImpl->setSubaddressLabel(accountIndex, addressIndex, label.toStdString());
 }
+void Wallet::deviceShowAddressAsync(quint32 accountIndex, quint32 addressIndex, const QString &paymentId)
+{
+    QFuture<void> future = QtConcurrent::run(m_walletImpl, &Monero::Wallet::deviceShowAddress,
```

Comment:
Could you please update the code to invoke async calls through `FutureScheduler`.
Ex.:
https://github.com/monero-project/monero-gui/blob/master/src/libwalletqt/Wallet.cpp#L465 (signal)
or
https://github.com/monero-project/monero-gui/blob/master/src/libwalletqt/Wallet.cpp#L642 (js callback)